pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -58,3 +64,35 @@ func PersistentVolumeClaimUsageFunc(object runtime.Object) api.ResourceList {
 	}
 	return result
 }
+
+// PersistentVolumeClaimConstraintsFunc verifies that all required resources are present on the claim
+// In addition, it validates that the resources are valid (i.e. requests < limits)
+func PersistentVolumeClaimConstraintsFunc(required []api.ResourceName, object runtime.Object) error {
+	pvc, ok := object.(*api.PersistentVolumeClaim)
+	if !ok {
+		return fmt.Errorf("unexpected input object %v", object)
+	}
+
+	// pvc level resources are often set during admission control
+	// As a consequence, we want to verify that resources are valid prior
+	// to ever charging quota prematurely in case they are not.
+	allErrs := field.ErrorList{}
+	allErrs = append(allErrs, validation.ValidatePersistentVolumeClaimSpec(&pvc.Spec, field.NewPath("spec"))...)
```

Comment:
> quota can be checked post-validation, but we are not there yet. The other concern, which I see is not actually checked in this PR (but I will add now that I noticed it) is that PVC requests for storage must be non-negative. A negative storage request would allow for gaming quota.

Let's fix the negative problem by preventing the admission plugin from decrementing usage.  An request decrementing usage is rejected.